client/src/components/EditProfile/EditProfileTab/EditProfileTab.tsx

```diff
@@ -0,0 +1,340 @@
+import useStyles from './useStyles';
+import Typography from '@material-ui/core/Typography';
+import { FormLabel, OutlinedInput, Select, MenuItem, TextField, Button, Box, Menu, Switch } from '@material-ui/core';
+import React, { useState } from 'react';
+import { useFormik } from 'formik';
+
+interface UserProfile {
+  firstName: string;
+  lastName: string;
+  gender: string;
+  birthdateMonth: string;
+  birthdateDay: string;
+  birthdateYear: string;
+  email: string;
+  phone: string;
+  address: string;
+  description: string;
+  accountType: string;
+  available: boolean;
+  availability: string;
+}
+
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+const days = [...Array(30).keys()].map((i) => i + 1);
+const years = [...Array(119).keys()].map((i) => i + 1903).sort((a, b) => b - a);
+
+export default function EditProfileTab(): JSX.Element {
+  const classes = useStyles();
+  const [accountType, setAccountType] = useState<UserProfile['accountType'] | null>('partner');
```

Comment:
Please note that eventually we won't need these states as we will be fetching the logged in user's profile (we can store it in a single state/in the context) 😄